doc/api/index.rst

```diff
@@ -16,6 +16,14 @@ Gravity Corrections
     normal_gravity
     bouguer_correction
 
+Gridding and Interpolation
```

Comment:
```suggestion
Equivalent Layers
```

The EQLs can do much more than interpolation: upward-continuation, data integration, reduction to the pole, derivatives, etc.